lib/actions.js

```diff
@@ -211,15 +211,16 @@ actions.validate = function(region, templateUrl, callback) {
 /**
  * Save a CloudFormation stack's configuration to a local file or S3
  *
- * @param {string} name - the base name of the stack (no suffix)
+ * @param {string} baseName - the base name of the stack (no suffix)
```

Comment:
After looking at the tests, `baseName` made sense to me, but generally I think I'm a bit confused by our usage of repoName and baseName for what similar things.